tpp-private/private.php

```diff
@@ -18,148 +18,139 @@
 </head>
 
 <body>
-<div class="container">
+    <div class="container">
 
-    <header>
-        <h1>Placeholder for actual header</h1>
-    </header>
+        <header>
+            <h1>Placeholder for actual header</h1>
+        </header>
 
```

Comment:
pls no inline style before we forget to take it out

Reply:
Sadly it breaks the code so I will try to figure out late